### ISSUE TITLE
libdeezer installation

### ISSUE THREAD

**Issue by rijnhard** [jonaski/strawberry#46]
i installed the latest deb for ubuntu xenial and it wouldn't load. 
Turns out it was looking for libdeezer. I eventually figured it out, but it should either ship with it, or have clear instructions on how to add it, or just not break if it's not installed.

for others, download the deezer sdk (link in readme) and then drop the appropriate `.so` into `/usr/lib`

for Mint 18 I copied this lib from the archive path `/deezer-native-sdk-v1.2.10/SDK/Bins/Platforms/Linux/x86_64/libdeezer.so`

**Comment by jonaski** [comment]
I only experimented with ubuntu xenial builder to build a AppImage that would include deezer, but I ran into to many issues. I gave up and just removed the builder.
The deezer library uses special debian/ubuntu-based so xenial was the last last release that worked with deezer. However it still works on Windows, that's why I kept the support.
See https://github.com/deezer/native-sdk-samples/issues/26 and https://github.com/jonaski/strawberry/issues/21

I didn't realize xenial was still supported, I'll add back the builder now. Unsure what to do about the deezer thing, I guess the best way would be to include the library as part of the .deb, but I think I have wasted enough time on it already to bother special-casing the debian files for xenial with deezer.
